tfhe/docs/fine_grained_api/quick_start.md

```diff
@@ -124,8 +124,8 @@ fn main() {
     let ct_3 = server_key.unchecked_add(&ct_1, &ct_2);
 
     // We use the client key to decrypt the output of the circuit:
-    let output = client_key.decrypt(&ct_3);
-    assert_eq!(output, (msg1 + msg2) % modulus as u64);
+    let output = client_key.decrypt_decode_padding(&ct_3);
```

Comment:
as indicated by Thomas for documentation I would keep the plain old decrypt as users should be using that in about all cases, applies to all md files and docstrings not related to decryption